rkt/image_rm.go

```diff
@@ -39,34 +39,51 @@ func rmImages(s *store.Store, images []string) error {
 	errors := 0
 	staleErrors := 0
 	for _, pkey := range images {
+		var keys []string
 		errors++
 		h, err := types.NewHash(pkey)
 		if err != nil {
-			stderr("rkt: wrong image ID %q: %v", pkey, err)
-			continue
-		}
-		key, err := s.ResolveKey(h.String())
-		if err != nil {
-			stderr("rkt: image ID %q not valid: %v", pkey, err)
-			continue
-		}
-		if key == "" {
-			stderr("rkt: image ID %q doesn't exist", pkey)
-			continue
+			var found bool
+			keys, found, err = s.ResolveName(pkey)
+			if len(keys) > 0 {
+				errors += len(keys) - 1
+			}
+			if !found {
+				stderr("rkt: image name %q not found", pkey)
+				continue
+			}
+			if err != nil {
+				stderr("rkt: image name %q not valid: %v", pkey, err)
```

Comment:
Is the image name really not valid? It could just be the database being corrupted somehow, and nothing to do with the image name `pkey`.

I suggest one of these:
```
stderr("rkt: cannot search image name %q: %v", pkey, err)
```
or just:
```
stderr("rkt: %v", err)
```